forc-plugins/forc-client/src/ops/run/op.rs

```diff
@@ -75,7 +75,7 @@ pub async fn run(command: RunCommand) -> Result<Vec<fuel_tx::Receipt>> {
         let inputs_to_add = locked_wallet
             .get_asset_inputs_for_amount(
                 AssetId::default(),
-                DEFAULT_SPENDABLE_COIN_AMOUNT,
+                1_000_000,
```

Comment:
I'm not sure about this. I've seen the whole thing removed in other places (mainly tests) but I don't know if that's safe to do here. cc @nfurfaro and @digorithm.

Reply:
Yeah, we've removed that `DEFAULT_SPENDABLE_COIN_AMOUNT` here https://github.com/FuelLabs/fuels-rs/pull/556.